src/sentry/runner/commands/presenters/webhookpresenter.py

```diff
@@ -48,9 +48,13 @@ def flush(self) -> None:
         ):
             return
 
-        region: Optional[str] = settings.SENTRY_REGION
-        if not region:
-            region = settings.CUSTOMER_ID
+        region: Optional[str] = (
+            settings.SENTRY_REGION
+            if settings.SENTRY_REGION
+            else settings.CUSTOMER_ID
+            if settings.CUSTOMER_ID
+            else settings.SILO_MODE
+        )
```

Comment:
Can I get some context on what this is representing?

Seeing `"region": "CONTROL"` in a JSON object might be a little confusing, relative to how we represent regions elsewhere. In a vacuum, `{"silo_mode": "CONTROL", "region": null, ...}` would be more consistent.